### DISCUSSION TITLE
error when running

### DISCUSSION THREAD

**Discussion by Rakan-Makhashin** [zh-plus/openlrc#15] [Q&A] [answered]
![image](https://github.com/zh-plus/openlrc/assets/112309932/93017da9-700e-4cf5-bb11-f138c4a5b9d0)
I am getting this error when running program

**Comment by zh-plus** [comment, marked as answer]
You should enclose your invocation code within the main function like this:

```python
if __name__ ==  '__main__':
    lrcer = LRCer()
    lrcer.run('./data/test.mp3', target_lang='zh-cn')
```

**Reply by VertofChest** [reply]
This did not help me at all.
